app/assets/src/components/common/CoverageVizSidebar/temporary_data/AE017283.1_coverage_viz.json

```diff
@@ -0,0 +1,2060 @@
+{
+  "total_length": 2560265,
+  "name": "Propionibacterium acnes KPA171202, complete genome",
+  "contigs": [],
+  "reads": [
+    [
+      "A00351:106:HH7CLDSXX:4:2356:13856:33505/1",
+      1535881,
+      1536030
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:2356:13856:33505/2",
+      1536121,
+      1535971
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:2161:5547:3521/1",
+      240015,
+      239922
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:2161:5547:3521/2",
+      239922,
+      240015
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:1130:22417:11256/1",
+      724879,
+      724801
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:1130:22417:11256/2",
+      724801,
+      724879
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:1426:31467:14497/1",
+      111390,
+      111475
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:1426:31467:14497/2",
+      111475,
+      111390
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:1305:5647:29058/1",
+      1216066,
+      1216216
+    ],
+    [
+      "A00351:106:HH7CLDSXX:4:1305:5647:29058/2",
+      1216524,
+      1216374
+    ]
+  ],
+  "coverage": [
+    0,
```

Comment:
I wonder if there is a more space-efficient representation of this ;)

Reply:
(I.e. since it’s so sparse, store only the indices and values that are actually non-zero?)

Reply:
Hey @cdebourcy . Yeah I can definitely store the indices. There are also situations where it's NOT sparse, so doing indices + values would be less efficient. I can add some logic for deciding which format to use based on # nonzero values. 

For the MVP, there are 1000 - 2000 bins max. I think we can defer this optimization until later, if we decide to handle more bins (maybe if we do zooming). Thoughts?

Reply:
This comment isn't within the scope of this PR (it's related to the idseq-dag change). Resolving for now, let's continue the convo.